dev/doc/release-process.md

```diff
@@ -28,7 +28,9 @@
 - [ ] For major releases, you can create new views in the above project (on the model of previous releases)
 - [ ] Pin the versions of libraries and plugins in [`dev/ci/ci-basic-overlay.sh`](../ci/ci-basic-overlay.sh) to use commit hashes. You can use the [`dev/tools/pin-ci.sh`](../tools/pin-ci.sh) script to do this semi-automatically.
 - [ ] In a PR on `master` to be backported, add a new link to the `'versions'` list of the refman (in `html_context` in [`doc/sphinx/conf.py`](../../doc/sphinx/conf.py)).
-- [ ] Add `{coq,coq-core,coq-stdlib,coqide-server}.X.X.dev` packages in [`core-dev`](https://github.com/coq/opam-coq-archive/tree/master/core-dev)
+- [ ] Add `{coq-core,rocq-init,coqide-server}.X.X.dev` packages in [`core-dev`](https://github.com/coq/opam-coq-archive/tree/master/core-dev)
+- [ ] Ensure a `coq-stdlib` package compatible with the new packages above exists either in [`ocaml repo`](https://github.com/ocaml/opam-repository) or in [`core-dev`](https://github.com/coq/opam-coq-archive/tree/master/core-dev)
```

Comment:
Why would a `coq-stdlib` package be released in opam-repository?

Reply:
In case a previous version of the stdlib keeps compiling with the new release for instance.

Reply:
Hum, right. I had forgotten that since the `coq` meta-package keeps being published to opam-repository, `coq-stdlib` has to do the same, in contradiction with the policy for most other Coq packages.